Aquality.Appium.Mobile.Template/Aquality.Appium.Mobile.Template.SpecFlow/Features/Demo.feature

```diff
@@ -0,0 +1,11 @@
+﻿Feature: Demo
+
+@demo
+Scenario: I try to login with invalid credentials
+	Given I can open 'Login Screen' view
+	Then Login Screen is opened
+	When I log in with data:
+	| Name     | Value           |
```

Comment:
I believe data tables should be shifted
https://cucumber.io/docs/gherkin/reference/#data-tables